### ISSUE TITLE
Files are displayed as folders (icon)

### ISSUE THREAD

**Issue by N1c093** [ciur/papermerge#26]
First of all great work :)

I just installed papermerge and noticed, that my files are displayed as folders. The icon seems to be wrong.
I installed it today via Docker with this documentation: https://papermerge.readthedocs.io/en/latest/setup/docker.html

Here you can see an example:

![image](https://user-images.githubusercontent.com/59510296/85781001-84541f80-b725-11ea-8b49-215db80d9cbf.png)

Do you need more information?

**Comment by ciur** [comment]
I know the problem. I will provide a fix in couple days. I think your Browser localization language is German. A quick workaround is to open application in English localized browser. I know it sounds weird. Bug is that decision to style node as folder or file is made by something like this:

  ```
  if node.human_name similar to  "document" :
      this is document
    else:
      this is folder
```
In case of German localization node.human_name is "Dokument" - which does not match word "Document". 
Anyway,  as I mentioned before I will fix code so that it won't depend on localization.

Thank you for your feedback!

**Comment by N1c093** [comment]
Thank you for your explanation :)

**Comment by ciur** [comment]
Fix is now available in v1.2.3. You basically need to: 
```
$ cd papermerge-proj/docker
$ docker-compose up -d
```
again. Docker compose will pull version v1.2.3 which includes above fix.
